roles/sap_hypervisor_node_preconfigure/tasks/platform/redhat_ocp_virt/install-nmstate-operator.yml

```diff
@@ -0,0 +1,89 @@
+---
+- name: Delete the nmstate operator namespace
```

Comment:
Why is the nmstate operator removed first?

Reply:
To ensure idem-potency